scripts/plot_operator_breakdown.py

```diff
@@ -76,7 +76,7 @@
 print(df)
 
 # Drop all operators that do not exceed 1% in any query
-df = df[df > 0.01].dropna(axis="columns", how="all")
+df = df[df >= 0.01].dropna(axis="columns", how="all")
```

Comment:
?

Reply:
Whooopsie. It really confused me that I needed to make this change twice. That explains it .... will revert.